changes-set.txt

```diff
@@ -88,10 +87,10 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+10-Jan-25 syl5bi    biimtrid
  9-Jan-25 fvresval  [same]      Moved from SF's mathbox to main set.mm
  9-Jan-25 brtp      [same]      Moved from SF's mathbox to main set.mm
  9-Jan-25 ---       ---         Moved initial surreal definitions from
-                                SF's mathbox to main set.mm
```

Comment:
not sure why this was deleted?

Reply:
Ooh good catch, thanks for noticing.

It was almost surely  the result of making a mistake when resolving git conflicts.